content/pt-br/docs/concepts/services-networking/network-policies.md

```diff
@@ -0,0 +1,348 @@
+---
+title: Políticas de rede
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+Se você deseja controlar o fluxo do tráfego de rede no nível do endereço IP ou de portas TCP e UDP 
+(camadas OSI 3 e 4) então você deve considerar usar Políticas de rede (`NetworkPolicies`) do Kubernetes para aplicações
+no seu cluster. `NetworkPolicy` é um objeto focado em aplicações/experiência do desenvolvedor 
+que permite especificar como é permitido a um {{< glossary_tooltip text="pod" term_id="pod">}} 
+comunicar-se com várias "entidades" de rede.
+
+As entidades que um Pod pode se comunicar são identificadas através de uma combinação dos 3 
+identificadores à seguir:
+
+1. Outros pods que são permitidos (exceção: um pod não pode bloquear a si próprio)
+2. Namespaces que são permitidos
+3. Blocos de IP (exceção: o tráfego de e para o nó que um Pod está executando sempre é permitido,
+independentemente do endereço IP do Pod ou do Nó)
+
+Quando definindo uma política de rede baseada em pod ou namespace, você utiliza um {{< glossary_tooltip text="selector" term_id="selector">}}
+para especificar qual tráfego é permitido de e para o(s) Pod(s) que correspondem ao seletor.
+
+Quando uma política de redes baseada em IP é criada, nós definimos a política baseada em blocos de IP (faixas CIDR).
+
+<!-- body -->
+## Pré requisitos
+
+As políticas de rede são implementadas pelo [plugin de redes](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/).  Para usar 
+uma política de redes, você deve usar uma solução de redes que suporte o objeto `NetworkPolicy`.
+A criação de um objeto `NetworkPolicy` sem um controlador que implemente essas regras não tem efeito.
+
+## Pods isolados e não isolados
+
+Por padrão, pods não são isolados; eles aceitam tráfego de qualquer origem.
+
+Os pods tornam-se isolados ao existir uma `NetworkPolicy` que selecione eles. Uma vez que
+exista qualquer `NetworkPolicy` no namespace selecionando um pod em específico, aquele pod 
+irá rejeitar qualquer conexão não permitida por qualquer `NetworkPolicy`. (Outros pod na mesma 
```

Comment:
Poderia ser: "Outros pods no mesmo namespace"

Reply:
Yes, sure! Thanks!
We need keep conversation in English, because the work group, they're may understand all conversation.